typegate/prod/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "typegate_prod"
```

Comment:
I would suggest to use the "standalone" as folder and "typegate" alone for the name of the crate